apps/dotcom/client/src/tla/components/TlaEditor/TlaLegacySnapshotEditor.tsx

```diff
@@ -28,47 +27,23 @@ export const components: TLComponents = {
 }
 
 export function TlaLegacySnapshotEditor({
-	fileSlug,
 	snapshot,
-	timeStamp,
-	context,
-	token,
+	fileSlug,
 }: {
```

Comment:
refactored this to only work on the old snapshot routes `/s/:slug` and made a new editor component specifically for the history snapshot.